DuckDuckGo/MainWindow/MainViewController.swift

```diff
@@ -564,6 +564,20 @@ extension MainViewController {
     }
 }
 
+// MARK: - BrowserTabViewControllerDelegate
+
+extension MainViewController: BrowserTabViewControllerDelegate {
+
+    func highlightFireButton() {
```

Comment:
Technically we could test this part. This involves change 

```
tabBarViewController = TabBarViewController.create(tabCollectionViewModel: tabCollectionViewModel)
```

to inject a factory to create `TabBarViewController.create ` that we can control in tests. Altough the ContextualOnboardingViewHighlighter should be injected as well in the TabBarViewController. Currently it has static methods.

Reply:
I assume you did not do it because it’s not a quick thing?

I think it’s very simple so probably not worth spending lots of time on this but if it’s easy why not?